mediathread/templates/assetmgr/install_safari_extension.html

```diff
@@ -1,26 +1,11 @@
 {% load staticfiles %}
 
-<div id="safari-install-container">
-    <h4><strong>Install Safari extension</strong></h4>
+<div id="firefox-install-container">
```

Comment:
This page is for the Safari extension, not Firefox. I think we wanted to update both Firefox & Safari pages to direct them to Chrome.